docs/features/externaltools/h5p.md

```diff
@@ -2,45 +2,65 @@
 
 # H5P 
 
-H5P is an open source library which delivers interactive html content. It can be embedded through iframes and has also integrations in different platform like moodle. Currently it is mainly used for educational content.
+H5P is an open source library which delivers interactive html content. It can be embedded through iframes and also offers integrations for different platforms like moodle. Currently, it is mainly used for educational content.
```

Comment:
```suggestion
H5P is an open source library which delivers interactive html content. It can be embedded through iframes and also offers integrations for different platforms (e.g. moodle). Currently, it is mainly used for educational content.
```